### DISCUSSION TITLE
Normal matrix layout in slideout

### DISCUSSION THREAD

**Discussion by wmdhosting** [craftcms/cms#11042] [Ideas] [answered]
![Snag_2aa986f3](https://user-images.githubusercontent.com/76176291/165892991-dfd96132-2230-49e7-9c8f-31e03a0722bd.png)

As slideout is default mode of work now. 
any chance to  put in there normal fields preview.. as in full page.
 much better than this preview mode..

**Comment by brandonkelly** [comment, marked as answer]
Already the case in Craft 4 :)

**Comment by wmdhosting** [comment]
ok thank you :)
